themes/godotengine/pages/fund/about.htm

```diff
@@ -0,0 +1,126 @@
+title = "Godot Development Fund | About"
+url = "/fund/about"
+layout = "fund"
+is_hidden = 0
+==
+
+{% put title %} Development Fund - About {% endput %}
+
+<h3 class="title">
+  Let’s make our dream engine together!
+</h3>
+<p>
+  <strong>Godot is the first Free and Open Source game engine in the same league
+  as the current big, commercial ones,</strong> with many innovations in
+  workflow, features and usability.
+</p>
+<p>
+  It offers a complete editor, dedicated 2D and 3D engines, animation tools,
+  multi-platform exports, plugins, a Free asset store, and so much more! All
+  that in a lightweight (30 MB) package that runs smoothly on Windows, Mac and
```

Comment:
30MB? Standard edition's download size maybe. I think that description needs to be clearer about what the size is referring to. Also note that the Mono version is larger due to its dependencies.

At the moment, that description makes it sound like the *whole thing* is 30MB installed. Which was never true.

Standard 3.2.3, with export templates, is about 670MB installed (with its binary weighing in at about 70MB of that on Linux). And while 670MB installed is over 20x the size of the 30MB download, it's still *considerably* smaller than either Unity or Unreal's installtion sizes.

You're always going to get trolls, but a more accurate description here is giving them less ammo (let's face it, they'll use *anything* as a reason to troll).

Reply:
I took the description as-is from the Patreon page.